packages/RecentChangesAPI/src/custom/modules/Users/clients/base/api/CustomRecentChangesApi.php

```diff
@@ -42,6 +43,7 @@ public function registerApiRest()
                 'shortHelp' => 'Identify Users who have had recent changes to their assigned records.',
                 'longHelp' => 'custom/modules/Users/clients/base/api/help/CustomRecentChangesApi.html',
             ),
+            //TODO: add a second listing here
```

Comment:
Can we create an issue to track doing this later?

Reply:
Opened  #14